ios/brave-ios/Sources/BraveWallet/Panels/Connect/EditSiteConnectionView.swift

```diff
@@ -162,15 +162,15 @@ struct EditSiteConnectionView: View {
             if sizeCategory.isAccessibilityCategory {
               VStack {
                 AddressView(address: account.address) {
-                  AccountView(address: account.address, name: account.name)
+                  AccountView(address: account.address, seed: account.address, name: account.name)
```

Comment:
should AccountView be parametrized by account instead of its fields?

Reply:
adding a new constructor that only takes an `AccountInfo`. the old constructer is still needed for places that don't have `AccountInfo` for example `SiteConnectionDetailView`